### ISSUE TITLE
From Owen Parsons on Dev Guides

### ISSUE THREAD

**Issue by jlivingood** [jlivingood/IETF-L4S-Deployment#32]
From:
Two queues are needed on the bottleneck link on an end-to-end path. 
To:
Two queues are needed on the bottleneck links, and on the end-to-end path. 

Sounds like link needs a plural?  Maybe also paths?

From:
In these cases, the peer using DSCP 45 for other purposes is responsible for remarking on ingree and egress from their network.
To
In these cases, the peer using DSCP 45 for other purposes is responsible for remarking on ingress and egress from their network.

From:
not a higher priority that other internet traffic. 
To:
not a higher priority than other internet traffic. 

Are there any ECN traffic simulators?  Methods to verify and test?

**Comment by chrisunderscorek** [comment]
> Are there any ECN traffic simulators?  

Yes, we use „JENS“ https://github.com/telekom/aml-jens (user land CLI) to apply a data rate pattern on an (linux) egress interface.

ECN markings (CE) are done based on queue sojourn time (to simulate an L4S aware-BBU of E///).
markings, sojourn times and bandwidth etc.  metrics are logged.  (by sch_janz kernel modul) https://github.com/tarent/sch_jens/tree/master/janz)
